docker-compose.yml

```diff
@@ -8,14 +8,18 @@ services:
     command: sh -c "rm -rf /build/* && cp -r /usr/share/nginx/html/* /build"
     container_name: ${COMPOSE_PROJECT_NAME-rsd}-admin
     image: ${COMPOSE_PROJECT_NAME-rsd}/admin
+    logging:
+      driver: "json-file"
+      options:
+        max-file: "1"
```

Comment:
I suggest using at least `max-file: "2"`. Otherwise I think Docker will throw away any old logs when reaching the 1MB limit. With `max-file: "2"` you'll always have at least the last 1MB of logs.

Reply:
done